### ISSUE TITLE
Merging of Affix text-reading breaking filters

### ISSUE THREAD

**Issue by zbee** [aeon0/d4lf#363]
My friend and I are both getting errors like these, where it is erroring out because it is reading 2 affixes as 1 - it does not happen every time.

<details><summary>Log Files and example item</summary>

```
2024-08-17 | 07:34:39.679 | MainThread | INFO | src.logger:102 | Running version v5.7.5
2024-08-17 | 07:34:39.679 | MainThread | INFO | main:31 | Adapt your configs via gui.bat or directly in: C:\Users\Pwnjo.d4lf
2024-08-17 | 07:34:39.680 | MainThread | ERROR | src.item.filter:285 | Could not load profile edited_e_end. Checked: C:\Users\Pwnjo.d4lf\profiles\edited_e_end.yaml
2024-08-17 | 07:34:39.690 | MainThread | INFO | src.item.filter:319 | Loading profile sigils: Sigils
2024-08-17 | 07:34:39.704 | MainThread | INFO | src.item.filter:319 | Loading profile c_end: Affixes Uniques
2024-08-17 | 07:34:39.714 | MainThread | INFO | src.item.filter:319 | Loading profile e_end: Affixes Uniques
2024-08-17 | 07:34:39.723 | MainThread | INFO | src.item.filter:319 | Loading profile c_spec: Affixes Uniques
2024-08-17 | 07:34:39.731 | MainThread | INFO | src.item.filter:319 | Loading profile e_spec: Affixes Uniques
2024-08-17 | 07:34:39.734 | MainThread | INFO | src.utils.window:73 | Using WinAPI to search for window: Diablo IV.exe
2024-08-17 | 07:55:23.068 | Thread-4 (_wrapper_run_loot_interaction_method) | INFO | src.loot_filter:149 | Run Loot filter
2024-08-17 | 07:55:23.471 | Thread-4 (_wrapper_run_loot_interaction_method) | INFO | src.loot_filter:36 | Items: 1 (favorite: 0, junk: 0) in Char_Inventory
2024-08-17 | 07:55:25.380 | Thread-4 (_wrapper_run_loot_interaction_method) | INFO | src.loot_filter:168 | Loot Filter done
2024-08-17 | 07:59:30.017 | Thread-5 (_wrapper_run_loot_interaction_method) | INFO | src.loot_filter:149 | Run Loot filter
2024-08-17 | 07:59:30.381 | Thread-5 (_wrapper_run_loot_interaction_method) | INFO | src.loot_filter:36 | Items: 1 (favorite: 0, junk: 0) in Char_Inventory
2024-08-17 | 07:59:31.757 | ThreadPoolExecutor-0_2 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='lucky hit chance damaging an elite enemy grants you a barrier absorbing up to damage for seconds. this effect can only happen once every seconds. »' ||| combined_lines='+4.0% lucky hit chance [4.0 - 5.0]% ( +4,0%) damaging an elite enemy grants you a barrier absorbing up to 1,333 [909 - 1,514] damage for 10 seconds. this effect can only happen once every 20 seconds. » empty socket operties lost when equipped:'
2024-08-17 | 07:59:32.127 | Thread-5 (_wrapper_run_loot_interaction_method) | INFO | src.loot_filter:168 | Loot Filter done
2024-08-17 | 08:00:49.714 | Thread-6 (vision_mode) | INFO | src.scripts.vision_mode:133 | Starting Vision Filter
2024-08-17 | 08:00:52.364 | ThreadPoolExecutor-0_0 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='lucky hit chance damaging an elite enemy grants you a barrier absorbing up to damage for seconds. this effect can only happen once every seconds. »' ||| combined_lines='+4.0% lucky hit chance [4.0 - 5.0]% ( +4,0%) damaging an elite enemy grants you a barrier absorbing up to 1,333 [909 - 1,514] damage for 10 seconds. this effect can only happen once every 20 seconds. » empty socket operties lost when equipped:'
2024-08-17 | 08:01:02.838 | ThreadPoolExecutor-0_4 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='lucky hit chance damaging an elite enemy grants you a barrier absorbing up to damage for seconds. this effect can only happen once every seconds. »' ||| combined_lines='+4.0% lucky hit chance [4.0 - 5.0]% ( +4,0%) damaging an elite enemy grants you a barrier absorbing up to 1,333 [909 - 1,514] damage for 10 seconds. this effect can only happen once every 20 seconds. » empty socket operties lost when equipped:'
```

![image](https://github.com/user-attachments/assets/442635b3-605e-435a-8e7f-0cbb8cd4cf5a)

```
2024-08-17 | 09:10:42.608 | MainThread | INFO | src.logger:102 | Running version v5.7.5
2024-08-17 | 09:10:42.609 | MainThread | INFO | main:31 | Adapt your configs via gui.bat or directly in: C:\Users\Jacob.d4lf
2024-08-17 | 09:10:42.635 | MainThread | INFO | src.item.filter:319 | Loading profile sigils: Sigils
2024-08-17 | 09:10:42.654 | MainThread | INFO | src.item.filter:319 | Loading profile c_end: Affixes Uniques
2024-08-17 | 09:10:42.673 | MainThread | INFO | src.item.filter:319 | Loading profile e_end: Affixes Uniques
2024-08-17 | 09:10:42.688 | MainThread | INFO | src.item.filter:319 | Loading profile c_spec: Affixes Uniques
2024-08-17 | 09:10:42.701 | MainThread | INFO | src.item.filter:319 | Loading profile e_spec: Affixes Uniques
2024-08-17 | 09:10:42.704 | MainThread | INFO | src.utils.window:73 | Using WinAPI to search for window: Diablo IV.exe
2024-08-17 | 09:10:56.756 | Thread-4 (_wrapper_run_loot_interaction_method) | INFO | src.loot_filter:149 | Run Loot filter
2024-08-17 | 09:10:59.159 | Thread-4 (_wrapper_run_loot_interaction_method) | INFO | src.loot_filter:36 | Items: 22 (favorite: 8, junk: 0) in Char_Inventory
2024-08-17 | 09:11:01.496 | ThreadPoolExecutor-0_1 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='lucky hit chance damaging an elite enemy grants you a barrier absorbing up to damage for seconds. this effect can only happen once every seconds.' ||| combined_lines='+4.0% lucky hit chance [4.0 - 5.0]% ( +4.0%) damaging an elite enemy grants you a barrier absorbing up to 1,333 [909 - 1,514] damage for 10 seconds. this effect can only happen once every 20 seconds. operties lost when equipped:'
2024-08-17 | 09:11:21.783 | Thread-4 (_wrapper_run_loot_interaction_method) | INFO | src.item.filter:111 | Matched e_end.Affixes.Boots: ['maximum_life', 'movement_speed', 'attacks_reduce_evades_cooldown_by_seconds']
2024-08-17 | 09:11:21.785 | Thread-4 (_wrapper_run_loot_interaction_method) | INFO | src.loot_filter:125 | Mark as favorite
2024-08-17 | 09:11:26.079 | ThreadPoolExecutor-0_2 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='mana per second while below life, evade freezes close enemies, deals cold damage to them, and makes them vulnerable for seconds.' ||| combined_lines='+2 mana per second +[2 - 3] (+2) while below 45% [45 - 52]% life, evade freezes close enemies, deals 6,015 cold damage to them, and makes them vulnerable for 3 seconds. operties lost when equipped:'
2024-08-17 | 09:11:30.320 | ThreadPoolExecutor-0_0 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='poison resistance when cast below maximum resource, basic skills generate of your primary resource, once per cast. when cast at or above maximum resource, basic skills deal increased damage. ,' ||| combined_lines='+24.0% poison resistance [24.0 - 32.0]% (+24.0%) when cast below 50% maximum resource, basic skills generate 3 of your primary resource, once per cast. when cast at or above 50% maximum resource, basic skills deal 40%[x] [20 - 68]% increased damage. , operties lost when equipped:'
2024-08-17 | 09:11:30.969 | Thread-4 (_wrapper_run_loot_interaction_method) | INFO | src.loot_filter:168 | Loot Filter done
2024-08-17 | 09:48:59.343 | Thread-5 (_wrapper_run_loot_interaction_method) | INFO | src.loot_filter:149 | Run Loot filter
2024-08-17 | 09:49:00.068 | Thread-5 (_wrapper_run_loot_interaction_method) | INFO | src.loot_filter:36 | Items: 22 (favorite: 0, junk: 0) in Char_Inventory
2024-08-17 | 09:49:02.039 | ThreadPoolExecutor-0_0 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='lucky hit chance damaging an elite enemy grants you a barrier absorbing up to damage for seconds. this effect can only happen once every seconds.' ||| combined_lines='+4.0% lucky hit chance [4.0 - 5.0]% ( +4.0%) damaging an elite enemy grants you a barrier absorbing up to 1,333 [909 - 1,514] damage for 10 seconds. this effect can only happen once every 20 seconds. operties lost when equipped:'
2024-08-17 | 09:49:02.045 | Thread-5 (_wrapper_run_loot_interaction_method) | INFO | src.item.filter:319 | Loading profile sigils: Sigils
2024-08-17 | 09:49:02.054 | Thread-5 (_wrapper_run_loot_interaction_method) | INFO | src.item.filter:319 | Loading profile c_end: Affixes Uniques
2024-08-17 | 09:49:02.061 | Thread-5 (_wrapper_run_loot_interaction_method) | INFO | src.item.filter:319 | Loading profile e_end: Affixes Uniques
2024-08-17 | 09:49:02.066 | Thread-5 (_wrapper_run_loot_interaction_method) | INFO | src.item.filter:319 | Loading profile c_spec: Affixes Uniques
2024-08-17 | 09:49:02.072 | Thread-5 (_wrapper_run_loot_interaction_method) | INFO | src.item.filter:319 | Loading profile e_spec: Affixes Uniques
2024-08-17 | 09:49:03.981 | ThreadPoolExecutor-0_1 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='to shocking impact after using teleport, close enemies are pulled to you and stunned for seconds, but teleports cooldown is increased by . ai al' ||| combined_lines="+1 to shocking impact [1 - 2] (+1) (sorcerer only) after using teleport, close enemies are pulled to you and stunned for 2.8 [2.0 - 3.0] seconds, but teleport's cooldown is increased by 20%[x]. (sorcerer only) ai al operties lost when equipped:"
2024-08-17 | 09:49:11.490 | ThreadPoolExecutor-0_5 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='staff @f endless cr q' ||| combined_lines=' staff @f endless cr q'
2024-08-17 | 09:49:11.496 | ThreadPoolExecutor-0_5 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='rage * ancestral unique staff' ||| combined_lines='rage * ancestral unique staff'
2024-08-17 | 09:49:11.506 | ThreadPoolExecutor-0_5 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='' ||| combined_lines='(sorcerer only)'
2024-08-17 | 09:49:11.512 | ThreadPoolExecutor-0_5 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='* to fireball * every rd cast of fireball launches additional projectiles and deals |x| increased damage. cn \ pr' ||| combined_lines='* +8 to fireball (+8) (sorcerer only) * every 3rd cast of fireball launches 2 additional projectiles and deals 86%|x| [50 - 90]% increased damage. (sorcerer only) cn \7 properties lost when equipped: ° ° ° chain lightning'
2024-08-17 | 09:49:11.514 | Thread-5 (_wrapper_run_loot_interaction_method) | INFO | src.item.filter:191 | Matched e_end.Uniques: staff_of_endless_rage
2024-08-17 | 09:49:11.516 | Thread-5 (_wrapper_run_loot_interaction_method) | INFO | src.loot_filter:125 | Mark as favorite
2024-08-17 | 09:49:17.324 | ThreadPoolExecutor-0_3 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='d@o@mibringer i , ancestral mythic unique sword' ||| combined_lines=' d@o@mibringer i) , ancestral mythic unique sword'
2024-08-17 | 09:49:17.328 | ThreadPoolExecutor-0_3 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='item power' ||| combined_lines='925 item power'
2024-08-17 | 09:49:20.936 | ThreadPoolExecutor-0_5 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='damage over time distant enemies have a chance to be stunned for seconds when they hit you. you deal increased damage to stunned or knocked down enemies.' ||| combined_lines='+50.0% damage over time [50.0 - 60.0]% ( ) distant enemies have a 10% chance to be stunned for 2 seconds when they hit you. you deal 18%[x] [15 - 30]% increased damage to stunned or knocked down enemies. operties lost when equipped:'
2024-08-17 | 09:49:32.392 | ThreadPoolExecutor-0_5 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='mana per second while below life, evade freezes close enemies, deals cold damage to them, and makes them vulnerable for seconds.' ||| combined_lines='+2 mana per second +[2 - 3] (+2) while below 45% [45 - 52]% life, evade freezes close enemies, deals 6,015 cold damage to them, and makes them vulnerable for 3 seconds. operties lost when equipped:'
2024-08-17 | 09:49:37.789 | ThreadPoolExecutor-0_2 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='to potent warding for each type of elemental damage you deal, gain | increased damage for seconds, up to | . dealing elemental damage refreshes all bonuses.' ||| combined_lines='+3 to potent warding [2 - 3] (+1) (sorcerer only) for each type of elemental damage you deal, gain 10.5%|[x] [10.0 - 20.0]% increased damage for 4 seconds, up to 42.0%|[x] [40.0 - 80.0]% . dealing elemental damage refreshes all bonuses. (sorcerer only) operties lost when equipped:'
2024-08-17 | 09:49:41.632 | ThreadPoolExecutor-0_0 | WARNING | src.item.descr.find_affixes:124 | Affix does not exist: cleaned_str='poison resistance when cast below maximum resource, basic skills generate of your primary resource, once per cast. when cast at or above maximum resource, basic skills deal increased damage. ,' ||| combined_lines='+24.0% poison resistance [24.0 - 32.0]% (+24.0%) when cast below 50% maximum resource, basic skills generate 3 of your primary resource, once per cast. when cast at or above 50% maximum resource, basic skills deal 40%[x] [20 - 68]% increased damage. , operties lost when equipped:'
2024-08-17 | 09:49:42.283 | Thread-5 (_wrapper_run_loot_interaction_method) | INFO | src.loot_filter:168 | Loot Filter done
```

</details>

**Comment by chrisoro** [comment]
Resolution? Font size? HDR? Upscaling?

**Comment by zbee** [comment]
- `3840 x 2160` and `3440 x 1440`
- Both Small font
- Both HDR disabled
- I have Nv DLSS > Quality, unsure about them

**Comment by chrisoro** [comment]
What resolution is the screenshot based on?

**Comment by zbee** [comment]
`3440 x 1440`

**Comment by chrisoro** [comment]
While I do know that we have some problems on multi line which I'll fix, I ran your image offline and it worked without any issues in 5.7.5.

**Comment by zbee** [comment]
I'll see if I can get some other screenshots that cause issues later today.

**Comment by chrisoro** [comment]
multi lines should be fixed with 5.7.6
